src/App.js

```diff
@@ -1,11 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.css';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
 
-function App() {
-  return (
-    <div className="App">
-    </div>
-  );
+class App extends Component {
+  state = {
+    center: [49.2580045, 4.0317044],
+    zoom: 14,
+  };
+
+  render() {
+    return (
+      <div>
+        <Map center={this.state.center} zoom={this.state.zoom}>
+          <TileLayer
+            attribution='&amp;copy <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+            url="https://maps.heigit.org/openmapsurfer/tiles/roads/webmercator/{z}/{x}/{y}.png"
```

Comment:
single (attribution) or double (url)  quotes, make your choice. What AirBnb says ?

Reply:
attribution deleted and corrected